CMakeLists.txt

```diff
@@ -113,6 +121,10 @@ install(TARGETS dinghycore
 	DESTINATION ${CMAKE_INSTALL_LIBDIR}
 	COMPONENT ${DINGHYCORE_COMPONENT}
 )
+install(TARGETS dinghywebextension
+	DESTINATION ${CMAKE_INSTALL_LIBDIR}/dinghy
+	COMPONENT "runtime"
```

Comment:
Not sure if the COMPONENT is right?

Reply:
Yes, `runtime` is the right call here :+1: